features/FxA-57-verify-reminder/README.md

```diff
@@ -0,0 +1,44 @@
+
+Verification Reminder Email Specification
+======================
+
+https://mozilla.aha.io/features/FXA-57
+
+As a new Firefox Accounts user,
+ I have signed up for an FxA account when **setting up sync**, but did not verify my email.
+ 
+ I should get 2 email reminders that will ask me to verify my email.
+ 
+ The first email arrives after 48 hours since sign up.
+ 
+ The second email arrives after 168 hours since sign up.
+ 
+## Basic Email Template
+
+> This is just a visual guide, please see detailed email content below
+
+![](basic-template.png)
+
+
+## Email 1
+
+### Header: Hello again.
+
+### Content:
+
+A few days ago you created a Firefox Account, but never verified it.
+Once verified, you can access your tabs, bookmarks, passwords and
```

Comment:
A connected device sounds to me like anything connected to the Internet. We should keep it about devices that are connected to the user's account.